### PR TITLE
run cargo update

### PR DESCRIPTION
**Pull request by ry**
Removes duplicated rustls crate

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -420,8 +435,8 @@ dependencies = [
  "termcolor",
  "test_util",
  "tokio",
- "tokio-rustls 0.14.0",
- "tokio-tungstenite",
+ "tokio-rustls",
+ "tokio-tungstenite 0.10.1",
```
Comment (bartlomieju) [inline Cargo.lock:439]: Two version of tungstenite?

File: Cargo.lock [hunk 2]
```diff
@@ -873,8 +907,8 @@ dependencies = [
  "bytes",
  "headers-core",
  "http",
- "mime 0.3.16",
- "sha-1",
+ "mime",
+ "sha-1 0.8.2",
```
Comment (bartlomieju) [inline Cargo.lock:911]: Ditto

File: Cargo.lock (hunk 1 above)
Comment (ry) [reply Cargo.lock]: fixed

File: Cargo.lock (hunk 2 above)
Comment (ry) [reply Cargo.lock:911]: can't fix... probably better than having two versions of rustls

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**